pinot-core/src/main/java/org/apache/pinot/core/segment/index/loader/defaultcolumn/BaseDefaultColumnHandler.java

```diff
@@ -303,6 +304,10 @@ protected void createColumnV1Indices(String column)
         dictionaryElementSize = StringUtil.encodeUtf8(stringDefaultValue).length;
         sortedArray = new String[]{stringDefaultValue};
         break;
+      case BYTES:
+        Preconditions.checkState(defaultValue instanceof byte[]);
+        sortedArray = new ByteArray[] {new ByteArray((byte[]) defaultValue)};
```

Comment:
The defaultValue is coming from JSON schema, correct? The expectation is to specify ` Hex.encodeHexString(bytes)`  in the schema. Which means that this code would need to use `Hex.decodeHex` here.

Also, if we don't have this documented, let's also add the expectation on how to specify the default value in the schema for BYTES.

Reply:
Actually the value is already of byte array type. We can document what/how to put a default value for byte column.